docs/CHANGELOG.rst

```diff
@@ -13,7 +13,11 @@ Unreleased
 
 Added
 -----
-- Add BWA-mem2 to the ``resolwebio/common`` Docker image
+- Add BWA-mem2 to the ``resolwebio/common`` Docker image 
```

Comment:
I think the trailing white space was added in the line 16. Please remove it. If you use Visual Studio Code as an editor, you can enable the `Trim Trailing Whitespace` option to assist you.